### DISCUSSION TITLE
Is it possible to render storybook as part of the frontend ?

### DISCUSSION THREAD

**Discussion by jesperordrup** [storybookjs/storybook#24694] [General]
The idea is to display stories in the frontend to let the visitors preview what is possible.

Is it possible in any way? With or without the storybook ui navigation? 

Im using sveltekit

**Comment by domyen** [comment]
Yes! Here are some ways:
- Publish Storybook to a public URL (I recommend chromatic.com)
- Iframe into a published Storybook

[Check these docs](https://storybook.js.org/docs/react/sharing/publish-storybook)

**Reply by rwcorbett** [reply]
I realize this is an older thread - but is there any way to do this WITHOUT an `<iframe>`? 

For example can we call a function to render the story as a static element? (I know that the `build-storybook` functionality exists and that renders all stories into a static HTML site... I would like to be able to render a SINGLE story this way)
